main.go

```diff
@@ -11,41 +15,64 @@ import (
 	. "github.com/env0/terratag/terraform"
 	. "github.com/env0/terratag/tfschema"
 	"github.com/env0/terratag/utils"
+
+	"github.com/hashicorp/go-hclog"
 	"github.com/hashicorp/hcl/v2/hclwrite"
-	"log"
-	"strings"
+	"github.com/hashicorp/logutils"
 )
 
 func main() {
-	Terratag()
-}
+	args, isMissingArg := InitArgs()
+	if isMissingArg {
+		return
+	}
+	filter := &logutils.LevelFilter{
+		Levels:   []logutils.LogLevel{"DEBUG", "TRACE", "INFO", "WARN", "ERROR"},
+		MinLevel: logutils.LogLevel("INFO"),
+		Writer:   os.Stderr,
+	}
+	if args.Verbose {
+		filter.MinLevel = logutils.LogLevel("DEBUG")
+	}
+	log.SetOutput(filter)
+	hclog.DefaultOutput = filter
```

Comment:
Just reiterating my comment on the previous closed PR - consider extracting this one to a function